superset-frontend/src/dashboard/components/nativeFilters/FilterBar/CascadeFilters/CascadePopover/index.tsx

```diff
@@ -61,7 +61,13 @@ const StyledTitle = styled.h4`
   padding: 0;
 `;
 
-const StyledIcon = styled(Icon)`
+const StyledEditIcon = styled(Icons.Edit)`
```

Comment:
Since we want the same styles for both icon types, it would probably be cleaner to use emotion's `css` style declarations instead of `styled`